### DISCUSSION TITLE
Support for round trips

### DISCUSSION THREAD

**Discussion by zell-mbc** [organicmaps/organicmaps#1258] [Q&A]
Hi there, let me start with saying that I really like the idea behind OrganicMaps, but I am not sure if my use case is what OrganicMaps is designed for.

I am trying to plan a hiking tour where start end end point are identical, with two stops on the way. I guess a typical round trip hiking tour with start and end at wherever I parked my car. 
But somehow I can't figure out how this is supposed to work. I would have thought I need to select start and destination and then add my stops one by one, but somehow this will only work when I artificially select a destination somewhat away from my starting point. If start and end are the same nothing happens.
Second thing I am unable to find is a way to save the route so I can quickly load it once at the starting point.

My first impression is that OrganicMaps is designed for ad-hoc usage, "show me the shortest way between my current position and wherever I need to go", rather than pre-planned trips? Which is fine, just means I may have to look for a different app.

**Comment by biodranik** [comment]
I also faced a similar problem :) How would you solve it in the simplest way for users? Now you may try to use a workaround: build some route and add one stop. Then zoom in and chose a point very close to your starting point and select "route to".

**Comment by zell-mbc** [comment]
Well, I was expecting that all I have to do is to select "create a route", select the same location as start and destination, and then add the real destination as a stop if a want to go back and forth on the same way. Or multiple stops along the way if I want to return via a different route.
Exactly what I ended up doing, but this only works when start and stop differ enough for OrganicMaps to accept it as a route. Pretty much what you wrote as well.
